### DISCUSSION TITLE
local develop serverless function with go

### DISCUSSION THREAD

**Discussion by wao3** [vercel/vercel#5079] [Help] [answered]
I followed the [tutorial of go](https://vercel.com/docs/serverless-functions/supported-languages#go) and run `vercel dev`, but when I visit the function(http://localhost:3000/api/date) it didn't work and throw an error. I have no idea to solve it and hope you can help me, thanks very much.
```
Command failed: go mod tidy
handler imports
	handler/Handler: package handler/Handler is not in GOROOT (C:\Go\src\handler\Handler)
```
ps. it worked when i deployed it to vercel

**Comment by styfle** [comment, marked as answer]
Hi @wao3 

Are you able to run `go mod tidy` locally without error?
